### ISSUE TITLE
Setting the StartWhenAvailable flag for Inline Scheduled Tasks

### ISSUE THREAD

**Issue by Kevinf63** [deadlydog/AzureDevOps.WindowsScheduledTasks#17]
Per the title, not sure how this can be done without using **XML** or **inline XML**, but I was wondering if the **inline** options UI/UX can be extended to include the **StartWhenAvailable** boolean flag; this "_Indicates that Task Scheduler can start the task at any time after its scheduled time has passed_", which is particularly useful for looping repetitive tasks.

**XML Scheduled Task Parameter:**
```
<Settings>
  <StartWhenAvailable>false</StartWhenAvailable>
</Settings>
```

**Documentation:**
https://docs.microsoft.com/en-us/powershell/module/scheduledtasks/new-scheduledtasksettingsset?view=windowsserver2019-ps

**Comment by deadlydog** [comment]
Yup, since there's a ScheduledTaskSettingsSet parameter for it (as per the link you provided), it should be possible to add that functionality. It would essentially be the same code as the `Enabled` inline option.

I don't foresee having time to get to this in the near future, but PRs are welcome :)

**Comment by deadlydog** [comment]
This is now available in v2.1.0 of the extension. Thanks @Kevinf63 !
